src/core/qgsproject.h

```diff
@@ -1269,6 +1298,26 @@ class CORE_EXPORT QgsProject : public QObject, public QgsExpressionContextGenera
     */
     void setPresetHomePath( const QString &path );
 
+    /**
+     * Registers the translatable containers into the translation context
+     * this is a rekursive function to get all the child containers
+     *
+     * \param translationContext where the objects will be registered
+     * \param parent parent-container containing list of children
+     * \param layerId to store under the correct context
+     * \since QGIS 3.4
+    */
+    void registerTranslatableContainers( QgsTranslationContext *translationContext, QgsAttributeEditorContainer *parent, const QString &layerId );
+
+    /**
+     * Registers the translatable objects into the translation context
```

Comment:
"Registers translatable objects into the translation context for inclusion in ts files."